package.json

```diff
@@ -38,7 +38,7 @@
   },
   "scripts": {
     "start": "node ./webpack.devserver.js",
-    "build": "NODE_ENV=production webpack",
+    "build": "NODE_ENV=production ./node_modules/.bin/webpack",
```

Comment:
Nope, `npm run` automatically adds `node_modules/.bin` to the path, so please revert that.